lib/diagnostic.rb

```diff
@@ -108,10 +115,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = { first_name: 'Amanda', favorite_number: 13}
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = person[:first_name] = 'Cindy'
```

Comment:
Super close here.

`'person.default = "#{person[:first_name]} is super cool"'`